front/components/navigation/NavigationSidebar.tsx

```diff
@@ -184,18 +184,38 @@ export const NavigationSidebar = React.forwardRef<
 
 function AppStatusBanner() {
   const { appStatus } = useAppStatus();
-  const { providerStatus } = appStatus ?? {};
 
-  if (!appStatus || !providerStatus) {
+  if (!appStatus) {
     return null;
   }
 
-  return (
-    <div className="space-y-2 border-y border-pink-200 bg-pink-100 px-3 py-3 text-xs text-pink-900">
-      <div className="font-bold">{providerStatus.name}</div>
-      <div className="font-normal">{providerStatus.description}</div>
-    </div>
-  );
+  const { providersStatus, dustStatus } = appStatus;
+
+  if (dustStatus) {
```

Comment:
So we can only display one banner at a time?

Reply:
dust has priority on providers. Only one at a time